first-network/scripts/script.sh

```diff
@@ -15,12 +15,14 @@ CC_SRC_LANGUAGE="$3"
 TIMEOUT="$4"
 VERBOSE="$5"
 NO_CHAINCODE="$6"
+NAME="$7"
```

Comment:
I think CC_NAME would be more descriptive and consistent with other variables like CC_SRC_LANGUAGE and while this adds an argument to this script specifying that argument requires editing byfn.sh, doesn't it?
On testing I noticed that a simple addition of that argument to byfn.sh doesn't even work because NO_CHAINCODE isn't set and the cc name argument is gobbled up as the NO_CHAINCODE argument.